src/components/TextArea.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <fieldset class="f-textArea">
+    <h4>
+      {{firstString}}
```

Comment:
because it is not the header, title or name, it is simply the predefined unchangeable text, that is displayed before the actual textarea. Would a renaming to prefix and suffix help?

Reply:
a header or name would suggest some kind of highlighting, but the text itself looks nearly identical to the text within the textarea

Reply:
yes, renaming them would help. `firstString`, `secondString` give no hints on what to expect from them

Reply:
renamed to label and footerLabel.